cxx-qt-lib/src/qpoint.rs

```diff
@@ -0,0 +1,70 @@
+// SPDX-FileCopyrightText: 2021 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use cxx::{type_id, ExternType};
+use std::mem::MaybeUninit;
+
+#[derive(Debug, Clone, Copy)]
+#[repr(C)]
+pub struct QPoint {
```

Comment:
Right, maybe something to discuss in #55 and #53 as you say this might be the more rust-y way. I think in this set which should aim for initial support with basic getter/setter.

Reply:
I think #53 Would be the appropriate issue here, as that issue deals with getters&setters.
I'll add a comment there.